tasks/util/validateTaskPlan.test.js

```diff
@@ -0,0 +1,36 @@
+import test from 'ava'
+import jsf from 'json-schema-faker'
+
+const Ajv = require('ajv')
+const { validateSchema } = require('feathers-hooks-common')
+const schema = require('../schemas/taskPlan')
```

Comment:
Hey @ahdinosaur Would love some help here as I've been working on this problem for a while with no luck.
I'm getting this error when running this test:
```
   [SyntaxError: Error resolving $ref pointer "http://localhost:3000/tasks/schemas/taskRecipe.json#/id". 
    Token "id" does not exist.]
```
Even when the [id property is in the schema](https://github.com/root-systems/cobuy/pull/124/files#diff-a00d73ceb321e9992c3b0b948dd25183R7)

Thanks!